content/v2.0/write-data/bulk-ingest-cloud.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Bulk ingest
+seotitle: Write bulk data to InfluxDB cloud.
+list_title: Write bulk data to InfluxDB cloud.
+weight: 105
+description: >
+  Write existing data to InfluxDB Cloud in bulk.
+menu:
+  v2_0:
+    name: Bulk ingest
+    parent: Write data
+products: [cloud]
+---
+
+The InfluxDB API is the way to feed ongoing collection of data into InfluxDB.
```

Comment:
Do we typically recommend that folks write streaming data to InfluxDB Cloud using the InfluxDB API?

Reply:
Might just get rid of this sentence.